spec/features/search/advanced_search/index_spec.rb

```diff
@@ -15,64 +15,11 @@
     expect(current_path).to eq('/search/advanced')
 
     expect(page).to have_content("Name")
-    expect(page).to have_content("City")
-    expect(page).to have_content("State")
+    expect(page).to have_content("Location")
     expect(page).to have_content("Age")
     expect(page).to have_content("Year")
-    expect(page).to have_content("Source")
 
     expect(page).to have_button("Search")
   end
 
-  it "I can see the results from the NYT API", :vcr do
```

Comment:
moved this test into another file since testing results on the advanced search started to seem weird once I was testing a bunch of different possibilities.